LogParser.cs

```diff
@@ -111,35 +130,52 @@ public LogParser(frmMain owner, string log)
         #region Methods
 
         public void LoadNamesTiers()
-		{
+        {
             logger.Debug("Begin Method: LogParser.LoadNamesTiers()");
 
-			namesTiers = new DataTable();
-			OleDbConnection dbConnect = null;
-			try 
-			{
-				dbConnect = new OleDbConnection("Provider=Microsoft.Jet.OLEDB.4.0;Data Source=" + owner.DBString);
-				string query = "Select NamesTiers.Name, NamesTiers.Tier, NamesTiers.TPercent, Sum(DKS.PTS) as SumOfPTS From (NamesTiers INNER JOIN DKS ON NamesTiers.Name = DKS.Name) GROUP BY NamesTiers.Name, NamesTiers.Tier, NamesTiers.TPercent";
-				OleDbDataAdapter dkpDA = new OleDbDataAdapter(query,dbConnect);
-				dbConnect.Open();
-				dkpDA.Fill(namesTiers);
-			}
-			catch (Exception ex) 
-			{
+            OleDbConnection dbConnect = null;
+
+            namesTiers = new DataTable();
+            try 
+            {
+                dbConnect = new OleDbConnection("Provider=Microsoft.Jet.OLEDB.4.0;Data Source=" + owner.DBString);
+                string query = "Select NamesTiers.Name, NamesTiers.Tier, NamesTiers.TPercent, Sum(DKS.PTS) as SumOfPTS From (NamesTiers INNER JOIN DKS ON NamesTiers.Name = DKS.Name) GROUP BY NamesTiers.Name, NamesTiers.Tier, NamesTiers.TPercent";
+                OleDbDataAdapter dkpDA = new OleDbDataAdapter(query,dbConnect);
+                dbConnect.Open();
+                dkpDA.Fill(namesTiers);
+            }
+            catch (Exception ex) 
+            {
                 logger.Error("Failed to load NamesTiers table: " + ex.Message);
-				MessageBox.Show("Could not open data connection. \n(" + ex.Message + ")","Error");
-			}
-			finally { dbConnect.Close(); }
+                MessageBox.Show("Could not open data connection. \n(" + ex.Message + ")","Error");
+            }
+            finally { dbConnect.Close(); }
+
+            alts = new DataTable();

```

Comment:
Had to add loading the alts table in to memory as well for this to work.  This through line 168 are new.